manual/manual/refman/exten.etex

```diff
@@ -1799,8 +1799,9 @@ quotes. For instance, you can use "[%sql {|...|}]" to
 represent arbitrary SQL statements -- assuming you have a ppx-rewriter
 that recognizes the "%sql" extension.
 
-Note that the non-extension form, for example "{sql|...|sql}", should
-not be used for this purpose. Indeed, the user cannot see in the code that
+Note that the word-delimited form, for example "{sql|...|sql}", should
+not be used for signaling an extension is in use.
+Indeed, the user cannot see in the code that
```

Comment:
Perhaps "know" rather than "see"?

Reply:
I think that `see` is better here, since with enough context, one can know that `{js|...|js}` is an implicit extension node.

Reply:
Hrm. Maybe then I misunderstand the point being made?

Reply:
Or I was unclear. What I mean is that with [%sql {|..|}], it is possible to see directly that this is an extension. With `{js|...|js}` it could be either a legitimate use of quoted strings, or some ppx stealing the quoted string syntax. I cannot see, locally, what is the expected semantics and I need to inquire about the current context; but I can still (hopefully) know. So I think that the immediacy of `see` is a good point here.

Reply:
This sounds better indeed, thanks!